tests/unit/objects/test_projects.py

```diff
@@ -98,19 +222,26 @@ def test_import_bitbucket_server(gl, resp_import_bitbucket_server):
     assert res["import_status"] == "scheduled"
 
 
-@pytest.mark.skip(reason="missing test")
-def test_list_user_projects(gl):
-    pass
+def test_list_user_projects(gl, resp_user_projects):
+    user_project = gl.users.get(1).projects.list()[0]
+    assert isinstance(user_project, UserProject)
+    assert user_project.name == "name"
+    assert user_project.id == 1
 
 
-@pytest.mark.skip(reason="missing test")
-def test_list_user_starred_projects(gl):
-    pass
+def test_list_user_starred_projects(gl, resp_starred_projects):
+    starred_projects = gl.users.get(1).starred_projects.list()[0]
+    assert isinstance(starred_projects, StarredProject)
+    assert starred_projects.name == "name"
+    assert starred_projects.id == 1
 
 
-@pytest.mark.skip(reason="missing test")
-def test_list_project_users(gl):
-    pass
+def test_list_project_users(gl, resp_list_users):
+    user = gl.projects.get(1).users.list()[0]
```

Comment:
Same as above, but for `project` fixture:

```suggestion
def test_list_project_users(project, resp_list_users):
    user = project.users.list()[0]
```